### PR TITLE
Modify base modal to use Semantic-UI modal base & lock deps

### LINKED ISSUE #348: Tests are failing for the application
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
Tests are failing for the application after recent commits. 

**Expected behavior:**
All tests must pass for the application.

Test for the PR:
https://www.codacy.com/app/fossasia/open-event-frontend/pullRequest?prid=723449

Tests after merge:
https://travis-ci.org/fossasia/open-event-frontend/builds/246608437?utm_source=github_status&utm_medium=notification

### PR DESCRIPTION
**Pull request by niranjan94**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #348

### CODE REVIEW

File: app/components/modals/modal-base.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import Ember from 'ember';
+import UiModel from 'semantic-ui-ember/components/ui-modal';
```
Comment (CosmicCoder96) [inline app/components/modals/modal-base.js]: @niranjan94 shouldn't this be `UiModal` , model seems mis leading.
Same in the commit message too.

Comment (niranjan94) [inline app/components/modals/modal-base.js]: Yep. Typo. Will correct once the build passes

**Comment by niranjan94** [comment]
I'll squash the commits at the end

**Comment by niranjan94** [comment]
@CosmicCoder96 @geekyd tests passed finally ! 😄 Commits squashed. Please review.

**Pull request merged**